module_intros/cluster.Cluster.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<p>Use the system (AABBQuery) that you already constructed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>